build.gradle.kts

```diff
@@ -17,8 +17,18 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-rootProject.group = "com.airthings.lib"
-rootProject.version = "0.2.6"
+val autoVersion = project.property(
+    if (project.hasProperty("AUTO_VERSION")) {
+        "AUTO_VERSION"
+    } else {
+        "LIBRARY_VERSION"
+    },
+) as String
+
+group = "${properties["GROUP"]}"
+version = autoVersion
```

Comment:
```suggestion
rootProject.group = "${properties["GROUP"]}"
rootProject.version = autoVersion
```

Reply:
I guess since we have only one project at the root, these two are effectively the same? But would need to test that this is true.

Reply:
In this case, yeah, but I like always to be very specific when setting the version and group. Just a nit :)

Reply:
Just not sure how KmmBridge behaves.

Reply:
I'll make the change and then we'll see :)